tasks/typescript/easy/index.ts

```diff
@@ -2,6 +2,7 @@
 
 function greet(name: string): string {
     // TODO: Add return statement
+    return name
```

Comment:
This function was supposed to return a string that prints "Hello, John Doe!". Right now it just returns the name, but that's fine.